solutions/cpython-tests/Makefile

```diff
@@ -56,7 +56,7 @@ ext2fs3.10: appdir3.10
 clean:
 	rm -fr appdir* ext2fs* @test_101* hostfs
 
-tests:
+run:
```

Comment:
I‘d prefer to run 3.10 for the PR pipeline because it gives us more test coverage, i.e., more passing cpython test cases. Is there any reason you prefer 3.9 over 3.10?